BuiltInSpicyAnalyzer.cmake

```diff
@@ -78,6 +78,12 @@ function (spicy_add_analyzer)
         if (SPICY_ROOT_DIR)
             target_include_directories(${lib} PRIVATE ${SPICY_ROOT_DIR}/include)
         endif ()
+
+        # Install Spicy grammars.
+        set(_SPIYC_SOURCES ${SPICY_ANALYZER_SOURCES})
+        list(FILTER _SPIYC_SOURCES INCLUDE REGEX "\.spicy$")
+        install(FILES ${_SPIYC_SOURCES} DESTINATION include/grammars/${NAME_LOWER})
```

Comment:
Not sure that's the right place, putting it under `include/` seems a bit confusing to me. Also, wouldn't they need to be inside a Spicy search path so that they can be imported?

Reply:
Not sure about the exact location either, another option would be somewhere under somewhere under `share/` (we already keep a Spicy `libdir` under `share/spicy/`.

My gut feeling was that we wouldn't want these installed into somewhere in the default Spicy search path so users do not pick them up accidentally.

Reply:
I switched over to installing into directory in Spicy's default search path under `share/`. I will open a separate docs PR to surface this.